--- drbd-utils-9.27.0/configure.ac.orig	2023-12-22 10:42:40.000000000 +0100
+++ drbd-utils-9.27.0/configure.ac	2023-12-28 22:28:00.048511920 +0100
@@ -422,18 +422,6 @@ else
     dnl default.
     test -z $INITDIR && INITDIR="$sysconfdir/init.d"
 
-    dnl Our udev rules file is known to work only with udev >= 85
-    if test "$WITH_UDEV" = "yes"; then
-       udev_version=$( set -- $($UDEVADM version); echo $1 )
-       if test -z "$udev_version"; then
-	    udev_version=$( set -- $($UDEVINFO -V); echo $3 )
-       fi
-       if test -z "$udev_version" || test "$udev_version" -lt 85; then
-          UDEV_RULE_SUFFIX=".disabled"
-          AC_MSG_WARN([Obsolete or unknown udev version. Installing disabled udev rules.])
-       fi
-    fi
-
     dnl Our sub-packages can be built for noarch, but RPM only supports
     dnl this from version 4.6.0 forward
     if test "$WITH_NOARCH_SUBPACKAGES" = "yes"; then
